subscription-engine/kafka-ws/app/StateManager.js

```diff
@@ -0,0 +1,13 @@
+const { ServiceStateManager } = require('@dojot/microservice-sdk');
+const { getConfig, transformObjectKeys } = require('@dojot/microservice-sdk/lib/configManager');
+const { camelCase } = require('lodash');
+
+class StateManager {
```

Comment:
Since you have a specialized `StateManager` class, why not create `signalReady` and `signalNotReady` methods for each of the objects to be monitored ('server', 'kafka' and 'redis')?

for instance:
signalReadyServer() and signalNotReadyServer()
signalReadyKafka() and signalNotReadyKafka()
signalReadyRedis() and signalNotReadyRedis()

otherwise, you don't need this class.

Reply:
Creating a function is necessary only for kafka, because it's necessary to register a function to check its state, the other "services" are ready on their way, that why I created this singleton and export it, like that these two services can use it as they want.